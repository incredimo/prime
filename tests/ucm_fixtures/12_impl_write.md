#!/usr/bin/env sh
grep -n "$1" "$2"