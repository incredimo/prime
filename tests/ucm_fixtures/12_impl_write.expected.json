{
  "id": "sT2",
  "verb": "set",
  "attrs": { "path": ".tools/loggrep/impl.sh", "mode": "write" }
}